regression/cbmc/cover-failed-assertions/test-no-failed-assertions.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+test.c
+--cover location --pointer-check --malloc-may-fail --malloc-fail-null
+\[main.coverage.4\] file test.c line \d+ function main block 4 \(lines test.c:main:12,13\): SATISFIED
+\[main.coverage.3\] file test.c line \d+ function main block 3 \(lines test.c:main:10\): FAILED
+\[main.coverage.2\] file test.c line \d+ function main block 2 \(lines test.c:main:4,5,7,9\): SATISFIED
+\[main.coverage.1\] file test.c line \d+ function main block 1 \(lines test.c:main:4\): SATISFIED
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
```

Comment:
:pick: It would be nice to have a comment in each of `.desc` files explaining what the test is intended to test. Even something pretty short could make this a while lot easier to maintain.